### ISSUE TITLE
Wrong Javadoc on Settings.renderKeywordStyle

### ISSUE THREAD

**Issue by DheerajKumarReddyKotha** [jOOQ/jOOQ#10442]
### Expected behavior and actual behavior:
**Expecting JavaDocs:**
```
/**
 * Whether the case of {@link org.jooq.Keyword} references should be modified in any way.
 * <p>
 * @deprecated - 3.12.0 - [#5909] - Use {@link RenderQuotedNames}, {@link RenderNameCase} and/or {@link RenderKeywordCase} instead.
 * /
@Deprecated
    public Settings withRenderKeywordStyle(RenderKeywordStyle value) {
        setRenderKeywordStyle(value);
        return this;
    }
```

**Actual JavaDocs:**
```
/**
     * Whether the case of {@link org.jooq.Keyword} references should be modified in any way.
     * <p>
     * @deprecated - 3.12.0 - [#5909] - Use {@link RenderQuotedNames} and {@link RenderNameCase} instead.
     * 
     */
    @Deprecated
    public Settings withRenderKeywordStyle(RenderKeywordStyle value) {
        setRenderKeywordStyle(value);
        return this;
    }
```
Following actual JavaDocs for replacement for `withRenderKeywordStyle()`, we used `withRenderNameCase(RenderNameCase.UPPER)` and that turned out to be not the correct implementation as it is turning the query to uppercase.

Figured out later that we should be using `withRenderKeywordCase(RenderKeywordCase.UPPER)` instead and then all the queries started executing fine.

**Additional Notes:**
Database: MySQL

**Comment by lukaseder** [comment]
Thanks a lot for your report. Makes sense, this looks like a copy paste error

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.14.0. Backport scheduled for 3.13.4 (#10446)
